### PR TITLE
check for "./scm-source.json" also

### PR DESCRIPTION
**Pull request by tuxlife**
@prayerslayer @hjacobs please comment

### CODE REVIEW

**Comment by prayerslayer** [comment]
We could read the entry name only once in a surrounding `let`, but that's a minor nitpick. Looks good to me.

**Comment by hjacobs** [comment]
@kolja might review too :smile:

File: src/org/zalando/stups/pierone/api_v1.clj [hunk 1]
```diff
@@ -236,7 +236,10 @@
                 tar-stream (TarArchiveInputStream. (GzipCompressorInputStream. fis))]
       (loop []
         (when-let [entry (.getNextTarEntry tar-stream)]
-          (if (= (.getName entry) "scm-source.json")
+          (if (or (= (.getName entry)
+                     "./scm-source.json")
+                  (= (.getName entry)
```
Comment (hjacobs) [inline src/org/zalando/stups/pierone/api_v1.clj]: I don't like that it's not "DRY" here...

Comment (kolja) [inline src/org/zalando/stups/pierone/api_v1.clj]: how about
(if-not (nil? (re-matches #"(.\/)?scm-source.json" (.getName entry))) (extract...) (recur))

Comment (hjacobs) [inline src/org/zalando/stups/pierone/api_v1.clj]: @kolja you can make it shorter ("nil" is false, i.e. "not nil?" is redundant)

**Comment by hjacobs** [comment]
:+1: (waiting for Travis CI build)

**Pull request merged**